### ISSUE TITLE
preset names with hypens not working

### ISSUE THREAD

**Issue by dimejo** [nils-werner/crestic#12]
I've previously (v0.3.0) been using preset names with hyphens in it like this: `home@workstation-local`. With crestic v0.4.0 I see this error:

```
user@workstation:~$ crestic home@workstation-local backup
usage: crestic [preset] command [arguments [arguments ...]]
crestic: error: argument preset: home@workstation-local is an invalid preset name, only preset names in the format of name[@suffix] are allowed.
```
I know this is probably caused by the new split preset feature, but is it intentional?

**Comment by nils-werner** [comment]
You're right, this was unintentional. I've pushed a fix and released 0.4.1

**Comment by dimejo** [comment]
Yep, that fixed it. Thank you so much for the crazy fast support!
